### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by snej** [cisco/mlspp#407]
I’m interested in using this, but I don’t see any documentation, not even header comments. Is there external documentation anywhere?

(I’ve been reading the OpenMLS docs, which is giving me an overview of how to use MLS, but of course the API is completely different.)

**Comment by bifurcation** [comment]
Yeah, this is definitely a gap, but I'm not sure when we'll get to fixing it.  As an interim solution, it might be helpful to look at the tests, especially in `state.cpp` and `session.cpp`, which provide examples for how to use most of the functionality.

**Comment by fluffy** [comment]
I hope we can up the priority of this one
